viewing_party/party.py

```diff
@@ -0,0 +1,129 @@
+#wave 1
+
+def create_movie(title, genre, rating):
+    if title ==  None:
+        return None
+    if genre == None:
+        return None
+    if rating == None:
+        return None
+    else:
+        movie = {"title": title, "genre": genre, "rating": rating}
+        return movie
+
+def add_to_watched(user_data, movie):
+    user_data['watched'].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data['watchlist'].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data['watchlist']:
+        if movie['title'] == title:
+            user_data['watchlist'].remove(movie)
+            user_data["watched"].append(movie)
+        if user_data['watched'] == []:
+            user_data["watched"].append(movie)
```

Comment:
so, if `user_data["watched"]` is empty, no worries! it will still append the appropriate movie from the `watchlist` to the `watched` list on line 26. So we don't need this extra checking if statement:

```suggestion

```